log4j2-elasticsearch-core/src/main/java/org/appenders/log4j2/elasticsearch/VirtualProperty.java

```diff
@@ -39,6 +40,20 @@ public VirtualProperty(final String name, final String value, final boolean isDy
         this.name = name;
         this.value = value;
         this.dynamic = isDynamic;
+        this.writeRaw = false;
```

Comment:
Let's call the new one here:

```java
this(name, value, isDynamic, false);
```

This change should also help with coverage report. If it doesn't, let's add an explicit test for it.

Reply:
I don't see how the old constructor can be called since we're always using a Builder to create new instances. The only place I find the old constructor used are in `org.appenders.log4j2.elasticsearch.hc.smoke.SmokeTest.createElasticsearchAppenderBuilder(boolean, boolean, boolean)` and `org.appenders.log4j2.elasticsearch.jest.smoke.SmokeTest.createElasticsearchAppenderBuilder(boolean, boolean, boolean)`

Reply:
It can be called by extending classes created by users. This applies to both `VirtualProperty` and `VirtualPropertyPlugin` The goal here is to ensure that they don't get compile errors once they upgrade to 1.6.

Reply:
Are you planning any updates on this? Basically, new test `constuctorWithNoWriteRawParamSetsWriteRawToFalse` that uses old constructor directly (no builder) should do the trick here.

```java

// given
// new VirtualProperty(String, String, boolean)

// then
// assert property.isWriteRaw() is false
```

Give me a shout if you need any help.